provisioning/common/etcd/values.yaml

```diff
@@ -266,11 +266,12 @@ extraDeploy:
                   {{- if .Values.defragmentation.cronjob.resources }}
                   resources: {{- toYaml .Values.defragmentation.cronjob.resources | nindent 16 }}
                   {{- end }}
-                    {{- if .Values.auth.client.secureTransport }}
+                  {{- if .Values.auth.client.secureTransport }}
+                  volumeMounts:
                     - name: certs
                       mountPath: /opt/bitnami/etcd/certs/client
                       readOnly: true
-                    {{- end }}
+                  {{- end }}
               volumes:
                 {{- if .Values.auth.client.secureTransport }}
                 - name: certs
```

Comment:
@zajca , @ dakraus send us the [PR](https://github.com/keboola/keboola-as-code/pull/1362) with fix for a typo, I copied it to make CI working. 

The error did not occur in our stacks, because we use an unencrypted connection (protected by network policy) and the `if` confition is evaluated as `false`.